packages/api/controllers/nomination.js

```diff
@@ -56,11 +57,19 @@ const updateNomination = async (req, res) => {
   try {
     const nomination = await db.Nomination.update(
       { status: req.body.status },
-      { where: { id } }
+      { where: { id }, returning: true }
     );
+    const updatedNom = nomination[1][0].dataValues
+    const updatedStatus = nomination[1][0].dataValues.status
```

Comment:
FYI you don't need this variable as you can just reference the property in your if statement below like this: 

```javascript
if (updatedNom.status === 'Decline') {
```